index.html

```diff
@@ -2747,6 +2747,26 @@ <h4>Context Injection</h4>
 (`https://www.w3.org/ns/did/v1`).
           </p>
 
+          <pre class="example nohighlight" title="A controller document without an @context property">
```

Comment:
```suggestion
          <pre class="example nohighlight" title="A controller document without a @context property">
```

Reply:
I generally read `@context` as "at-context", not as "context", partly because it is distinct from `context`. That reading makes `an @context` appropriate, not `a context`. Perhaps others should weigh in, so we can decide which indefinite article to use throughout.

Reply:
@jandrieu — Thanks for that sleuthing. https://github.com/w3c/json-ld-syntax/pull/448 created to make that errant `a` match the other six `an`.

Reply:
I also read it as "an at-context"

Reply:
Yes, the Editors have attempted to consistently read `@context` as `at context` and therefore try to keep a consistent rule of us `an` vs. `a`.